source/js/components/petition/petition.jsx

```diff
@@ -261,31 +259,6 @@ export default class Petition extends React.Component {
     });
   }
 
-  /**
-   * sign the user up for the mozilla newsletter.
-   *
-   * @returns {promise} the result the XHR post attempt.
-   */
-  signUpToBasket() {
-    this.setState({ basketSubmitted: true });
```

Comment:
I think `basketSubmitted` can be removed from other parts of the code too?

Reply:
good point